packages/office-ui-fabric-react/src/components/HoverCard/ExpandingCard.base.tsx

```diff
@@ -76,12 +76,13 @@ export class ExpandingCardBase extends BaseComponent<IExpandingCardProps, IExpan
   private _onRenderExpandedCard = (): JSX.Element => {
     // firstFrameRendered helps in initially setting height of expanded card to 1px, even if
     // mode prop is set to ExpandingCardMode.expanded on first render. This is to make sure transition animation takes place.
-    !this.state.firstFrameRendered &&
```

Comment:
@atneik you were the one who worked initially on this implementation and I know this line was important on making the animation work correctly.

Reply:
With this change, we are setting `firstFrameRendered` state as `true` async, even if it was already `true`. I'm trying to understand how is this fixing the issue?

Reply:
Moreover, this is causing an infinite loop as long as the card is open.

Reply:
I have been able to repro the issue pretty consistently based on window size. I think it's something that should be fixed. Do you know of a bitter fix @atneik ?